src/style.css

```diff
@@ -0,0 +1,77 @@
+/* stylelint-disable font-family-no-missing-generic-family-keyword */
+@font-face {
+  font-family: "Avenir";
+  src: url(/fonts/Avenir.ttc);
+  font-weight: normal;
+  font-style: normal;
+}
+@font-face {
+  font-family: "Avenir";
+  src: url(/fonts/Avenir-Book.ttf);
+  font-weight: 300;
+  font-style: normal;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url(/fonts/Avenir-Heavy.ttf);
+  font-weight: 500;
+  font-style: normal;
+}
+
+.form {
+  display: flex;
+  position: relative;
+  margin: 20px auto;
+  box-sizing: border-box;
+  background-color: #fff;
+}
+
+.form__input {
+  height: 66px;
+  width: 100%;
+  padding-left: 62px;
+  font-size: 16px;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+  color: #3d4e61;
+  font-family: Avenir;
```

Comment:
why do we need to use `font-family: Avenir` a lot of times? Add this to `html` classes